### PR TITLE
Bump until new version

### PR DESCRIPTION
**Pull request by stephr0y**

### CODE REVIEW

File: bin/cli.js [hunk 1]
```diff
@@ -14,7 +14,13 @@ function getParentBranch() {
 }
 
 function bump(type) {
-    return execSync(`npm version ${type}`);
+    const packageName = execSync(`cut -d "=" -f 2 <<< $(npm run env | grep "npm_package_name")`, {encoding: 'utf8'}).trim();
```
Comment (GermainBergeron) [inline bin/cli.js]: You can do `npm info . name` to get the name of the project in the current folder, I don't know if it's ok for your use case

In the same idea (if it works), you could probably remove this line and do `npm show . versions -json`

File: bin/cli.js [hunk 2]
```diff
@@ -14,7 +14,13 @@ function getParentBranch() {
...
+    const publishedVersions = JSON.stringify(execSync(`npm show ${packageName} versions -json`, {encoding: 'utf8'}));
+    let newVersion = execSync(`npm version ${type}`, {encoding: 'utf8'}).trim().substr(1);
```
Comment (rousselm4) [inline bin/cli.js]: Un do-while serait pas pire ici

**Review by rousselm4 — APPROVED** [review]

**Review by GermainBergeron — APPROVED** [review]
Ignore my comment if it doesn't work

**Review by pochretien — APPROVED** [review]

File: bin/cli.js (hunk 1 above)
Comment (stephr0y) [reply bin/cli.js]: Nice, I looked a while for this. I'll change it.

**Review by GermainBergeron — APPROVED** [review]

**Pull request merged**